.github/workflows/build.yml

```diff
@@ -137,16 +143,20 @@ jobs:
           docker compose exec app test/test.sh
 
   push:
-    runs-on: ubuntu-24.04
+    runs-on: ubuntu-latest
     needs:
       - merge
       - test
     env:
       DOCKER_APP_IMAGE: ${{ needs.merge.outputs.image }}
+      DOCKER_APP_IMAGE_ARM64: ${{ needs.merge.outputs.build-image-arm64 }}
+      DOCKER_APP_IMAGE_X64: ${{ needs.merge.outputs.build-image-x64 }}
     steps:
       - name: Checkout code
         uses: actions/checkout@v4
 
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
```

Comment:
Hm, I don't know if we want a newline here since the other steps all have that spacing.